doc/smart-contract-upgrades.md

```diff
@@ -0,0 +1,194 @@
+# Smart Contract Upgrades
+
+Code on the EVM is immutable but via proxies, smart contracts can be “upgraded”. A proxy contract acts as an
```

Comment:
I think the documentation should be little more specific to our contracts - the `_Example:_` section should have an example of how one should proceed if they need to upgrade `LightClient.sol`. Otherwise I think, this becomes general documentation about smart contract upgradeability which can be found online. What are your thoughts on the same?

Reply:
fair point, i'll add more info specific to LightClient.sol

Reply:
sorted here https://github.com/EspressoSystems/espresso-sequencer/commit/50dec6911459403d8c662ae7ac331fb00730888e